action.yml

```diff
@@ -59,9 +63,10 @@ runs:
         env:
           TAILSCALE_AUTHKEY: ${{ inputs.authkey }}
           ADDITIONAL_ARGS: ${{ inputs.args }}
+          ADDITIONAL_DAEMON_ARGS: ${{ inputs.daemon_args }}
           HOSTNAME: ${{ inputs.hostname }}
         run: |
-          sudo tailscaled 2>~/tailscaled.log &
+          sudo tailscaled ${ADDITIONAL_DAEMON_ARGS} 2>~/tailscaled.log &
```

Comment:
There are several conflicts with subsequent changes already in `main` on this line, will need to rebase as GitHub cannot do it automatically.